src/test/java/BaseTest.java

```diff
@@ -0,0 +1,42 @@
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.annotations.*;
+
+public class BaseTest {
+
+    protected WebDriver driver;
+    ChromeOptions options = new ChromeOptions();
+
+    boolean headless = false;
+
+    @BeforeMethod
+    public void init() {
+        WebDriverManager.chromedriver().setup();
+        this.driver = new ChromeDriver();
+
+        driver = new ChromeDriver(options);
+        if(headless) {
+            driver.manage().window().setSize(new Dimension(1920, 1200));
+        } else {
+            driver.manage().window().maximize();
+        }
+    }
```

Comment:
You don't use `Session` and it's okay, but think about passing driver for creating each page object class. It's inconveniently